xml/operations-maintenance-deploy-ptf.xml

```diff
@@ -24,28 +24,17 @@
     When &suse; has developed a PTF, you will receive a URL for that PTF. It
     will contain the entire Python virtual environment (venv). You should
     download the packages from the location provided by &suse; Support to a
-    temporary location on the &clm;. Set the variables for your Support ID and
-    password. For example:
+    temporary location on the &clm;. For example with a &o_netw; PTF:
    </para>
-   <screen>&prompt.ardana;sudo user=<replaceable>USER_NAME</replaceable>
-&prompt.ardana;sudo psw=<replaceable>PASSWORD</replaceable>
+<screen>&prompt.ardana;mkdir ~/PTF
+&prompt.ardana;cd ~/PTF
 &prompt.ardana;sudo wget --no-directories --recursive --reject "index.html*"\
---user=<replaceable>USER_NAME</replaceable> \
---password=<replaceable>PASSWORD</replaceable> \
---no-parent https://ptf.suse.com/5d0981855eecb9f040ed3a0fff8af236/cloud8/16374/x86_64/20181030/venv-openstack-neutron-x86_64-11.0.2-13.8.1.16374.0.PTF.1113308.noarch.rpm</screen>
+--no-parent --ask- password --user=USER_NAME \
+https://ptf.suse.com/5d0981855eecb9f040ed3a0fff8af236/cloud8/16374/x86_64/20181030/venv-openstack-neutron-x86_64-11.0.2-13.8.1.16374.0.PTF.1113308.noarch.rpm</screen>
```

Comment:
do we really want to document an existing path here? especially one tha tpoints to a real customer incident rather than some fake "this is for demo purposes" PTF? I would like to encourage to hide this by obfuscating the url.